README.md

```diff
@@ -68,7 +68,7 @@ Read our [contributing guide](https://reactjs.org/contributing/how-to-contribute
 
 ### Good First Issues
 
-To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/react/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. This is a great place to get started.
+To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/react/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. This is a great place to start when looking to contribute to React for the first time.
```

Comment:
I think it unnecessarily repeats the beginning wording "get you familiar". Also text "This is a great place to get started." is more to me an encouragement to take action and the paragraph is short enough to not summarise what one just have read.